Added available_template option to template sensor.

Limych opened the pull request:
**Description:**

Sometimes when using the template sensor it is very necessary to get the state of its inaccessibility.
For example, if you use the template sensor as a source in the utility meter, when the source data for the template sensor is temporarily unavailable, the calculation in the utility meter is seriously lost (because at this moment the value of the template sensor becomes zero for an instant).

To work correctly with such cases, I added a template for the status of sensor availability. I am sure it will also be useful in many other cases. For example, if you need to temporarily ignore some values ​​in further calculations inside HA.

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html


<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10130"><img src="https://gitpod.io/api/apps/github/pbs/github.com/Limych/home-assistant.io.git/b521a83742454f7ba3051baa278d56a061fb170e.svg" /></a>



<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10130"><img src="https://gitpod.io/api/apps/github/pbs/github.com/Limych/home-assistant.io.git/297b9db52286feed32854150f4ff8f60432bb9fe.svg" /></a>



<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10130"><img src="https://gitpod.io/api/apps/github/pbs/github.com/Limych/home-assistant.io.git/297b9db52286feed32854150f4ff8f60432bb9fe.svg" /></a>

farmio committed:
KNX component updates for xknx 0.11 (#9682)

* xknx 0.11

* update for xknx 0.11.1

- state_address instead of address
- sync_state option

* :pencil2: Tweak

- Please check if this config variable `device_class` belongs here. For now it breaks netlify with building.

* copy and past error -.-

* even more typos

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweaks

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweaks

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweak

* :pencil2: Tweak

patrickeasters committed:
Fix typo for Fortinet Developer Network URL (#10073)

cogneato committed:
fix google calendar docs link

google calendar docs linking to wrong page

cogneato committed:
fix google calendar docs link

google calendar docs linking to wrong page

corneels committed:
Remove password requirement from documentation (#10082)

With a recent change Google Passwords are no longer required. https://github.com/home-assistant/home-assistant/pull/25316

Removed reference to password requirement from documentation.

atomicpapa committed:
config example incorrect (#10083)

changed ip_address to host as in docs below

michaeldavie committed:
Typo in environment_canada (#10080)

gtdiehl committed:
Update initial HA Release from 0.96 to 0.97 (#10078)

Sensor Integration for the Rainforest Eagle-200 did not make it into 0.96.  The initial release was 0.97.

bigwoof committed:
remove password from configuration options (#10076)

Update documentation to reflect the recent change in 0.97.0 release

adonno committed:
replace address with state_address (#10084)

as per 0.97 release notes xKNX climate now uses : state_address instead of address

SNoof85 committed:
Missing web links (#10063)

Santobert committed:
replace operating_mode with hvac_mode (#10075)

DaveCo1701 committed:
Scene support for Zooz Z-Wave Switches (#10067)

Added instructions on how to update the switch and openzwave config file to add scene support for Zooz Zen26 & Zen27 switches.

pvizeli committed:
Update docker.markdown

pvizeli committed:
Update docker.markdown

pvizeli committed:
Update docker.markdown

Jackenmen committed:
Update installation.markdown (#10091)

balloob committed:
0.97.1

maxxer committed:
Replace port(s) -> pin(s) (#10095)

To be verified, but should be correct

ptarjan committed:
Add Fulfillment section

Without this being filled in, that oauth flow wouldn't start for me. I found it in this video: https://www.youtube.com/watch?v=MMtCzX6US9M

DubhAd committed:
Merge pull request #10104 from ptarjan/patch-1

Add Fulfillment section

csjo committed:
Update fortigate.markdown (#10111)

balloob committed:
Release 97.2

klaasnicolaas committed:
✏️ Add more information to Deconz (#10112)

* :pencil2: Add more information to Deconz

* :pencil2: Improve text about cover devices

derandiunddasbo committed:
Remove initial_state attributes from the examples (#10119)

Recently I had an extensive discussion about the pros and cons of using the `initial_state` attribute in automations and in the end I had to admit, that there are good arguments against using it unless absolutely necessary and only if one really knows what he or she is doing.

In the examples above I can't see a good reason for using `initial_state` and thus I'd propose to delete it from both examples.

...and they were inkonsistent in both examples anyways.  ;-) One of them used 'on' and the other used true.

tofuSCHNITZEL committed:
Update webhooks.markdown (#10122)

updated default trusted ips

tomlut committed:
Corrected template examples (#10114)

Template examples were incorrectly quoted.

AlecRust committed:
Mention integration in ZHA docs (#10108)

Due to the previous wording of this I thought a `zha` entry was necessary in `configuration.yaml`, which wasn't working for me on Hass.io. Adding ZHA from the Integrations page did work, but this isn't mentioned in the docs.

ties committed:
Document the buienradar camera component (#10106)

* Document the buienradar camera component

Buienradar camera had been added in 0.95 but documentation was not
added.

* :pencil2: Tweak

ogarling committed:
prefix HTTP scheme in example base_url (#10118)

* prefixed HTTP protocol in example base_url

* :pencil2: Tweak

DubhAd committed:
Added config check for Docker (#10100)

Added the config check steps for Docker

nielstron committed:
Add note about language issues with syncthru (#10109)

* Add note about language issues with syncthru

* :pencil2: Tweak

aetaric committed:
Update HaveIBeenPwned to include API Key (#9977)

* Update HaveIBeenPwned to include API Key

* Update haveibeenpwned.markdown

* :ambulance: Small Markdown fix

* :pencil2: Tweak

JeffLIrion committed:
Add info about customizing entity_picture for a person (#10120)

* Add info about customizing entity_picture for a person

* Add ':', change JPG to jpg

* Add link to the customizing entities page

Limych committed:
Added available_template option to template sensor.

Limych committed:
Added available_template option to template sensor.

DubhAd committed:
Clarity edit about `websocket-client` (#10129)

The constant references to installing the Python package causes confusion for Hass.io and Docker users, breaking these out.

emilgil committed:
Update twitter.markdown (#10128)

Twitter has new regulations in place regarding their developer accounts and API access, all applications for dev-accounts are handled manually which can take several weeks even if the application is filled in correctly.

jonathanweinberg committed:
small typo (#10132)

lowercase `F` for `false`

jbrody17 committed:
Thermostat sensor changed from hvac_mode to hvac_state (#10133)

huangda1982 committed:
Update aquostv.markdown (#10135)

ebaschiera committed:
fix quotes in example code (#10142)

evansj committed:
Add note about BASE_URL format (#10143)

Hopefully others can learn from my frustration. 😄

grillp commented:
Hey @Limych , I already have a PR open for teh same feature, but I added it for all template components: https://github.com/home-assistant/home-assistant/pull/25891 & https://github.com/home-assistant/home-assistant.io/pull/10123

amelchio committed:
Add netgear_lte.disconnect_lte (#10144)

balloob committed:
Add HA Cast promo

DubhAd committed:
Markdown fix (#10154)

* Markdown fix

Fixing the markdown

* Fix

Tediore committed:
Add note about purge_keep_days (#10152)

The purge_keep_days variable in the recorder component limits the max duration of a history stats sensor. This might not be clear to some users.

balloob committed:
Update PR template (#10147)

Jc2k committed:
homekit_controller updates + troubleshooting (#10153)

* homekit_controller updates

* Minor changes

kpine committed:
Remove update_config from zwave services (#10139)

abmantis committed:
edp_redy: remove component (#10145)

The service has moved to a new portal and the previous API does not work
anymore.

tcoenraad committed:
Add OTGW domestic hot water enable option as service (#10105)

akasma74 committed:
add record service to the list of services (#10137)

and rearranged services alphabetically

balloob committed:
Document ecobee attributes (#10146)

achatham committed:
Document Lutron Powr Savr occupancy sensor support (#10110)

* Document Lutron Powr Savr occupancy sensor support

* Minor change

zhumuht committed:
add retry option description for broadlink (#10150)

* add retry option description for broadlink

* Add default

atomicpapa committed:
Notification of non working status

Added text at the beginning to let folks know that this integration is currently not working

klaasnicolaas committed:
:pencil2: Tweak

DubhAd committed:
Merge pull request #10155 from atomicpapa/patch-11

Notification of non working status

Commit:
Add python3-dev in Raspberry Pi installation commands (#10046)

Commit:
Add python3-dev in Raspberry Pi installation commands (#10046)

majuss committed:
Fixed elv / pca 301 (#9995)

* fixed elv/pca

* fixed indent

hdsheena committed:
Slight changes to wording on customization help page (#10057)

* Update customizing-devices.markdown

* :pencil2: Tweak

* Minor changes

hdsheena committed:
Slight changes to wording on customization help page (#10057)

* Update customizing-devices.markdown

* :pencil2: Tweak

* Minor changes

frenck committed:
Revert "Notification of non working status" (#10158)

dshokouhi committed:
Correct typo in service call (#10160)

lyghtnox committed:
Update domain (#10164)

I messed up when I submitted the updated doc for snapcast. Sorry about that, here is the correct one with the snapcast domain

balloob committed:
Fix note end

zhumuht committed:
add support for yeelight: Yeelight Serene Eye-Friendly Desk Lamp (#10159)

* add support for yeelight: Yeelight Serene Eye-Friendly Desk Lamp (YLTD03YL) . product url: https://www.yeelight.com/en_US/product/muse
it is can use model 'mono', tested pass.

* :pencil2: Tweak

Commit:
Update joaoapps_join.markdown (#10167)

scop committed:
Document homematic default local IP (#10170)

ptarjan committed:
Update to required (#10174)

If you don't have this you get:

```
Invalid config for [roku]: required key not provided @ data['roku'][0]['host']. Got None. (See ?, line ?). Please check the docs at https://home-assistant.io/components/roku/
```

DubhAd committed:
Fixing indents and formatting (#10176)

Too many indents, and some extra spaces

abmantis committed:
Delete edp_redy.png (#10182)

This follows https://github.com/home-assistant/home-assistant.io/pull/10145

ludeeus committed:
Remove the documentation for the ruter integration (#10169)

* Remove the documentation for the ruter integration

* Remove image

dannerph committed:
Added documentation for the keba-charging-station component (#9616)

* Added documentation for the keba-charging-station component

* Update source/_components/keba.markdown

push version number to 0.95

Co-Authored-By: Klaas Schoute <klaas_schoute@hotmail.com>

* added refresh_interval documentation and service call examples

* bumped ha_release to 0.96, hopefully ;)

* :pencil2: Tweak

- Remove linkable title (no longer needed)
- Put service name in the title/header

* updated documentation for new services

* small addition

* Update source/_components/keba.markdown

Co-Authored-By: Martin Hjelmare <marhje52@kth.se>

* Update source/_components/keba.markdown

Co-Authored-By: Martin Hjelmare <marhje52@kth.se>

* Update source/_components/keba.markdown

Co-Authored-By: Martin Hjelmare <marhje52@kth.se>

* Update source/_components/keba.markdown

Co-Authored-By: Martin Hjelmare <marhje52@kth.se>

* :pencil2: Tweak

* :pencil2: Fix link

* :pencil2: Last tweak

jshridha committed:
Dialogflow v2 documentation (#10149)

* Dialogflow v2 documentation

* :pencil2: Tweak

* :pencil2: Tweak url

vondk committed:
Change of notification icon placement (#10187)

Change of notification icon placement => Just updated to new placement

atxbyea committed:
Update tensorflow.markdown (#10183)

* Update tensorflow.markdown

Update with examples of scan interval and callng the scanning process based on state from an entity

* :pencil2: Tweak

klaasnicolaas committed:
🚧 Add remove warning for edp_redy integration (#10178)

TychoWerner committed:
Added warning for integration getting removed (#10168)

* Added warning for integration getting removed

The googlehome integration is getting removed so letting users know when they wish to setup they know it's going away according to this pull: https://github.com/home-assistant/home-assistant/pull/26035

* :pencil2: Tweak

cj-thornton committed:
Add path option to SABnzbd component docs (#10124)

klaasnicolaas committed:
🔨 Fix broken links (part 5) (#10181)

* 🔨 Fix broken links (part 5)

* Update name

balloob committed:
Update frontend.markdown

ludeeus committed:
Removes the documentation for the googlehome integration (#10166)

* Removes the documentation for the googlehome integration

* remove image

tiagofreire-pt committed:
Add mention for TX-55CX680B (#10191)

doudz committed:
Add zigate to configuration variable (#10194)

dependabot[bot] committed:
Bump nokogiri from 1.10.3 to 1.10.4 (#10190)

Bumps [nokogiri](https://github.com/sparklemotion/nokogiri) from 1.10.3 to 1.10.4.
- [Release notes](https://github.com/sparklemotion/nokogiri/releases)
- [Changelog](https://github.com/sparklemotion/nokogiri/blob/master/CHANGELOG.md)
- [Commits](https://github.com/sparklemotion/nokogiri/compare/v1.10.3...v1.10.4)

Signed-off-by: dependabot[bot] <support@github.com>

tkislan committed:
Add minio documentation (#9267)

* Add minio documentation

* Update minio.markdown

* Update minio.markdown

* Review changes

* :pencil2: Tweak

* Update minio.markdown

* :pencil2: Tweak

* :pencil2: Tweak

kellerza committed:
Update sma.markdown (#10163)

* Update sma.markdown

* Update sma.markdown

* :pencil2: Tweaks

pvizeli committed:
Remove vagrant (#10192)

* Remove vagrant

* Remove references and unused asset

d0ugal committed:
Update climate.markdown (#10199)

JeffLIrion committed:
Add descriptive fields to script documentation (#10180)

* Add descriptive fields to script documentation

* Remove comment

* Add descriptions of configuration variables

* Working on documenting configuration variables

PDXPill committed:
Update utility_meter.markdown (#10175)

Hi, 

I was adding the last part of this page to my Home Assistant installation. 
Icon was not allowed, and the value must between quotes.

michaeldavie committed:
Correct specification for radar site IDs (#10148)

* Correct specification for radar site IDs

* :pencil2: Tweak

mickdekkers committed:
Update references to away mode in generic_thermostat documentation (#10171)

* Update references to away mode in generic_thermostat documentation

* :ambulance: Fix build error

klaasnicolaas committed:
✨ Troubleshooting automation page back in menu (#10185)

akasma74 committed:
added Testing section (#9767)

* added Testing section

I saw no such thing in docs and it constantly causes topics like this - https://community.home-assistant.io/t/condition-problems/124222.
Hope this addition is useful.

* make it more readable

* moving Testing section to this better place

as discussed here https://github.com/home-assistant/home-assistant.io/pull/9767#issuecomment-511128560

* moving Testing bit to Troubleshooting

as discussed here https://github.com/home-assistant/home-assistant.io/pull/9767#issuecomment-511128560

* :pencil2: Tweak

* :ambulance: Removing linkable_title

danner26 committed:
Update nest.markdown (#9757)

I added a warning alerting new users that Works With Nest Developer accounts can no longer be created, and they need a Works With Google account to setup a custom integration for Nest products.

gadgetchnnel committed:
Updated documentation for template binary_sensor (#9106)

* Updated documentation for template binary_sensor

* Update binary_sensor.template.markdown

* Updated section header

thomasloven committed:
Documentation for home-assistant/home-assistant-polymer#3505 (#10200)

* Documentation for home-assistant/home-assistant-polymer#3505

* Add raw tags

thomasloven committed:
Remove dead links

tiagofreire-pt committed:
Update commandline.markdown (#10201)

* Update commandline.markdown

On Hassio 0.97.2.

* Update commandline.markdown

balloob committed:
Merge branch 'current' into next

balloob committed:
Merge branch 'next' into rc

balloob committed:
Add draft blog post

1DontEx1st committed:
Update androidtv.markdown (#10205)

updated for latest FireTV OS

frenck committed:
Upgrade to Jekyll 4.0.0 (#10202)

filcole committed:
Nissan Leaf API changes. Remove device tracker support.

filcole committed:
Nissan Leaf API changes. Remove device tracker support.

JeffLIrion committed:
Document user-provided services.yaml file (#10186)

* Document user-provided services.yaml file

* :pencil2: Tweak

JeffLIrion committed:
Document user-provided services.yaml file (#10186)

* Document user-provided services.yaml file

* :pencil2: Tweak

kpine committed:
Remove reference to removed zwave update_config (#10210)

* Update installation.markdown

Remove reference to `update_config` service, which has been removed.

* :ambulance: Fix build error

kpine committed:
Remove reference to removed zwave update_config (#10210)

* Update installation.markdown

Remove reference to `update_config` service, which has been removed.

* :ambulance: Fix build error

balloob committed:
Merge remote-tracking branch 'origin/current' into rc

balloob committed:
update draft

balloob committed:
Update in other news

balloob committed:
More release notes updates

balloob committed:
Add components picture

balloob committed:
metion unifi + deconz having options

balloob committed:
Add alerts

balloob committed:
Tweak alerts text

balloob committed:
Add HA Alerts to footer

balloob committed:
Update things

gerard33 committed:
With hass.io no need for additional packages

Installed the Fritz!Box device tracker today on Hass.io without the need to install additional packages.

balloob committed:
Add alert custom element to integrations

balloob committed:
Add alert custom element to integrations

balloob committed:
remove need for css

balloob committed:
remove need for css

balloob committed:
Fix formatting.

balloob committed:
Fix formatting.

GoSpursGoNL committed:
Update heos.markdown

DubhAd committed:
Merge pull request #10207 from filcole/nissanleaf-login-fix

Nissan Leaf API changes. Remove device tracker support.

DubhAd committed:
Merge pull request #10215 from gerard33/patch-4

With hass.io no need for additional packages

DubhAd committed:
Merge pull request #10216 from GoSpursGoNL/patch-2

Update heos.markdown

Matt-PMCT committed:
Clarify device blocking functionality (#10217)

* Clarify device blocking functionality

The documentation did not clearly explain that you had to use MAC addresses to allow for blocking/unblocking clients.

* Minor change

frenck committed:
Remove GitPod badge from PR's (#10206)

onlymejosh committed:
Update sonos_say copy (#10220)

I noticed a mistake in the TTS engine example it should be `tts.google_translate_say` not `tts.google_say`

OnFreund committed:
CoolMaster: Change auto to heat_cool (#10209)

OnFreund committed:
CoolMaster: Change auto to heat_cool (#10209)

Derkades committed:
Remove hashes in front of commands (#10219)

On the website they look like commented lines

tubalainen committed:
small typo (#10234)

Small typo in the "full example" part. :)

vocweb committed:
Update condition (#10239)

condition: numeric state => condition: numeric_state

fabaff committed:
Add new source

pvmil committed:
Minor update time pattern trigger in web-interface (#10226)

I spent some time (newbie) before I found out I had to fill '*' for fields I did not plan to fill (like 'hour')

KennethLavrsen committed:
Document 'name' config parameter in rejseplanen (#10243)

* Document 'name' config parameter in rejseplanen

Document the undocumented but fully working name config. This is an optional config but essential to know to define more than one instance of the rejseplanen sensor and getting a predictable result. Otherwise the sensor name defaults to sensor.next_departure_# and the numbers can swap around after an HA restart

* An example where policy and common sense clash

Done as suggested though I do not agree

* :pencil2: Tweak

* Adjust default

* Fix sample

atxbyea committed:
Update from Integration to component (#10233)

* Update from Integration to component

* Revert a change

DubhAd committed:
It's a template! (#10245)

The example isn't valid, oops

pvizeli committed:
Update docker.markdown

cogneato committed:
Adds Breaking Changes descriptions

Add breaking changes bling

cogneato committed:
fix adr-400 link

fix adr-400 link in breaking changes section

balloob committed:
Add latest notes

balloob committed:
Update post

balloob committed:
Merge branch 'rc' into current

pvizeli committed:
Update HassOS 3.4

thomasloven committed:
Typo in local URL (#10252)

Commit:
Update plugwise.markdown (#10253)

minor typo in full configuration example

h4ckninja committed:
Fix dead link (#10247)

cogneato committed:
Changed heading to be a link (#10254)

* Changed heading to be a link

Changed "To prepare your machine for the Hass.io installation, run the following commands:" to a linkable heading so it can be shared more easily

* Add a section title

fortepc committed:
Updated phrasing (#10246)

* Updated phrasing

Updating phrasing of information to make things more clear and easy to understand.

* Minor changes

* Move the content to a section

atxbyea committed:
Information about notification (#10232)

* Information about notification

The pairing info now appears as a lovelace notification, not a state popupcard.

* Update location

balloob committed:
Release 98.1

genestealer committed:
Update hp_ilo.markdown (#10223)

* Update hp_ilo.markdown

* Move period

balloob committed:
Update notes

h4ckninja committed:
Minor typo fix (#10255)

AlecRust committed:
Include HA version in update available example (#10257)

* Include HA version in update available example

* :ambulance: Fix build error

jkeljo committed:
Update docs for new alarmdecoder keypress feature (#10195)

* Update docs for new alarmdecoder keypress feature

* Update service name after feedback on the parent PR

bbrendon committed:
new media_player  media_content_type (#10259)

Commit:
Added an extra note regarding data consumption (#10261)

I got stuck trying to figure out why my HASS install was consuming gigs of data each month only to realize that this component was running every few hours

DubhAd committed:
Fixing errors (#10267)

There's a couple of examples where `.state` should be appended, but isn't

hasscasts committed:
Quick typo "id" -> "is" (#10268)

* Quick typo "id" -> "is"

I believe this is a typo and so thought I'd give it a quick fix

* :pencil2: Fix spelling

balloob committed:
0.98.2

haakonstorm committed:
Update counter.markdown (#10273)

* Update counter.markdown

Added details for getting counters to work when counting errors, with example configuration.

* :pencil2: Tweak

* :pencil2: Tweak

Commit:
Update documentation for Configuration UI (#10266)

Server Control moved to its own section rather than General.  Add missing sections and descriptions.  Reorder to match UI.

gadgetchnnel committed:
New template sensor attributes docs (#10204)

* Updated documentation for template sensor with attributes

* Update template.markdown

Fix documentation

alemuro committed:
Add BeeWi SmartClim BLE sensor documentation (#10222)

* Add BeeWi SmartClim BLE sensor documentation

* Apply requested comments

* Remove non supported option 'monitored_conditions'

fmartens committed:
Add inverted RFLink cover description (#10177)

* Add inverted RFLink cover description

Added a section about the KAKU ASUN-650 (an 'inverted' RFLink cover) to the RFLink cover documentation.

* Update source/_components/cover.rflink.markdown

Co-Authored-By: Klaas Schoute <klaas_schoute@hotmail.com>

* Reformated the section about the type property

* Added the type property to the configuration section

* :pencil2: Tweak

* :pencil2: Tweak

* Shorten the type property description

* :pencil2: Tweak

* Update cover.rflink.markdown

* Remove 'default:' from the type property.

* :pencil2: Tweak

JeffLIrion committed:
Remove 'stopped' as a valid state for 'state_detection_rules' (#10212)

* Document user-provided services.yaml file (#10186)

* Document user-provided services.yaml file

* :pencil2: Tweak

* Remove 'stopped' as a valid state for 'state_detection_rules'

Rocik committed:
Update Supla's page with new device supported (#10265)

frenck committed:
Fixes updater notification template (#10278)

frenck committed:
Removes linkable_title occurance (#10276)

Commit:
MQTT Sensor: Warn about synchronization/double trigger issues. (#10262)

* Update sensor.mqtt.markdown

Add a warning in the documentation that the state and attributes of a sensor will not change synchronously if they share the same MQTT topic and automations may be triggered twice.  Reference closed issues #21699, and #21082.

* Update sensor.mqtt.markdown

formatting.

* Update sensor.mqtt.markdown

formatting.

* Fix anchor

* No double space after period.

* :pencil2: Tweak

orb9 committed:
Update ifttt.markdown (#10280)

* Update ifttt.markdown

* Update format

costastf committed:
Update google_maps.markdown (#10279)

Documentation updated to match change in https://github.com/home-assistant/home-assistant/pull/26361

frenck committed:
Full commonmark + general upgrades (#10275)

Kdemontf committed:
Update remote_rpi_gpio.markdown (#10274)

* Update remote_rpi_gpio.markdown

* :pencil2: Tweak

* Update remote_rpi_gpio.markdown

* Update remote_rpi_gpio.markdown

* :pencil2: Tweaks

cereal2nd committed:
Cert expiry documentation for config entries (#10241)

* Velbus quality scale

* Update cert expiry doc for #25624

* Revert "Velbus quality scale"

This reverts commit f972eb9e21d332737002364b91bc8ce8ce26329c.

StephenWetzel committed:
Add documentation for two new service methods (#10235)

* Add documentation for two new service methods

* Change method name to match latest change in PR

MatthewFlamm committed:
Add nws weather. (#9372)

* Add nws weather.

* Add station default description

* Add link to pynws library

* Remove double spaces

* :truck: Rename file

* Update to 0.96

Co-Authored-By: Klaas Schoute <klaas_schoute@hotmail.com>

* Update release_version and fix station log level.

It was requested in the backend PR to change the log level from INFO to DEBUG

* Add explanation for no logo

* Bump to 0.98.  Standardize punctuation and quotes

* Remove logo line entirely.

* v0.99

* :pencil2: Update to our latest requirements

* :ambulance: Fix build error

* :pencil2: Tweaks

frenck committed:
Adds logo to NWS (#10281)

bramkragten committed:
Lovelace background theme (#10283)

* Remove background from examples

* Update views.markdown

frenck committed:
Removes NOOP linkable_title plugin (#10282)

exxamalte committed:
GeoNet NZ Quakes Sensor (#10229)

* documents sensor

* :pencil2: Tweak

ualex73 committed:
Added documentation for new Slide cover integration (#10127)

* Added GoSlide documentation

* Renamed GoSlide to Slide, because of Innovation in Motion rebranding

* Removed timeout/retry configuration options.

* Updated version 0.98 to 0.99

iantrich committed:
Update glance.markdown (#10287)

* Update glance.markdown

https://github.com/home-assistant/home-assistant-polymer/pull/3584

* Add default

maxxer committed:
typo in persistent_notification.markdown (#10291)

pvizeli committed:
0.98.3

Quentame committed:
Add config flow to linky (#10284)

* Add config flow to linky

* :pencil2: Tweak

johnluetke committed:
Updates pi_hole component documentation per home-assistant/home-assistant#25837 (#10097)

frenck committed:
Merge branch 'current' into next

scop committed:
UpCloud URL tweaks (#10294)

kellerza committed:
Update device_sun_light_trigger.markdown - add person (#10237)

* Update device_sun_light_trigger.markdown

* :pencil2: Tweak

zewelor committed:
Add info about nightlight switch (#10242)

* Add info about nightlight switch

* :pencil2: Tweak

brange committed:
Update tellstick.markdown (#10196)

Add information about protocol and model for named sensors

oischinger committed:
Add ViCare component documentation (#10211)

* Create vicare.markdown

* Add viessmann.png for vicare component

* Update vicare.markdown

More documentation on services

* Update vicare.markdown

Added missing configuration parameter. Thanks @klaasnicolaas

* Document Water Heater component

* :pencil2: Tweak

* Apply suggestions from code review

Co-Authored-By: Martin Hjelmare <marhje52@kth.se>

* Mention all keyword in service calls

* Add missing all keyword reference

* :pencil2: Tweaks

vaidyasr committed:
Updated documentation for hostname template (#10113)

Limych committed:
Merge remote-tracking branch 'origin/feature/template-available' into feature/template-available

# Conflicts:
#	source/_components/template.markdown

klaasnicolaas commented:
This looks not good 🤔

frenck reviewed (changes requested):
@Limych there are a lot of unrelated commits in your PR.
Could you please clean this up? Also, it is targeting the `next` branch, does this mean it has a parent PR in the Home Assistant code base? If so, please link it in the opening post. When the PR is not related to a PR in the Home Assistant code base, please change the targeted branch to `current`.
